### ISSUE TITLE
Upload the compiled assets to our CDN

### ISSUE THREAD

**Issue by simonrjones** [w3c/w3c-website#38]
**Describe the issue**
We need a single, consistent location to load the design system static assets for the w3.org site and others. 

@deniak has recommended:

> After giving more thoughts to that topic, the npm package might not be necessary and will add more complexity to the process (version management, etc).
> We also expect most people to simply use the different components without having to deal with SASS or to have some knowledge of npm or composer.
> In the end, it'll just be simpler to upload the compiled assets to our CDN so that people can just link to these and start using the components in their pages.
> We don't need to keep different versions available and the CDN will always be up-to-date with the latest revision of the repository under the following URL:
> https://cdn.w3.org/assets/website-2021/{styles,js,svg,images,fonts}/
> 
> We are thinking about building a GitHub action to take care of validating the different assets, upload them to the CDN and invalidate the cache.

**Solution**
1. Work on  GitHub action to copy static assets into the CDN once code is merged into main. 
2. Update [getting started](https://github.com/w3c/w3c-website-templates-bundle/blob/main/docs/getting-started.md) docs on how users can use the design system assets outside of Symfony.

**Additional context**
See Basecamp thread on this https://3.basecamp.com/3091560/buckets/15695452/messages/4121857311
